projects/gloo/cli/pkg/cmd/version/clients.go

```diff
@@ -113,18 +114,92 @@ func (k *kube) Get(ctx context.Context) ([]*version.ServerVersion, error) {
 	return []*version.ServerVersion{serverVersion}, nil
 }
 
+func (k *kube) GetClusterVersion() (*version.KubernetesClusterVersion, error) {
+	cfg, err := kubeutils.GetConfig("", "")
+	if k.kubeContext != "" {
+		cfg, err = kubeutils.GetConfigWithContext("", "", k.kubeContext)
+	}
```

Comment:
These lines feel redundant. `GetConfig` just calls `GetConfigWithContext` and passes an empty string, so I think we can just simplify this to:
```suggestion
	cfg, err = kubeutils.GetConfigWithContext("", "", k.kubeContext)
```

Reply:
i assume this was probably left for backwards compatibility or something and never refactored. i can remove it.